sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/ModelSettingsSamples.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
+using static Azure.IoT.TimeSeriesInsights.Samples.SampleLogger;
+
+namespace Azure.IoT.TimeSeriesInsights.Samples
+{
+    internal class ModelSettingsSamples
+    {
+        /// <summary>
+        /// This sample demonstrates getting Time Series model settings, updating a model and changing the default type Id for a model.
+        /// </summary>
+        public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
+        {
+            PrintHeader("TIME SERIES INSIGHTS MODEL SETTINGS SAMPLE");
+
+            #region Snippet:TimeSeriesInsightsSampleGetModelSettings
+            Response<TimeSeriesModelSettings> getModelSettingsResponse = await client.ModelSettings.GetAsync();
+            Console.WriteLine($"Retrieved Time Series Insights model settings:\n{JsonSerializer.Serialize(getModelSettingsResponse.Value)}");
+            #endregion
+
+            // Store the default type Id so it can be used during clean up
+            string defaultTypeId = getModelSettingsResponse.Value.DefaultTypeId;
+
+            #region Snippet:TimeSeriesInsightsSampleUpdateModelSettingsName
+            Response<TimeSeriesModelSettings> updateModelSettingsNameResponse = await client.ModelSettings.UpdateNameAsync("NewModelSettingsName");
+            Console.WriteLine($"Updated Time Series Insights model settings name:\n" +
+                $"{JsonSerializer.Serialize(updateModelSettingsNameResponse.Value)}");
+            #endregion
+
+            // Create a Time Series type
+            var aggregateVariable = new AggregateVariable(new TimeSeriesExpression("count()"));
```

Comment:
I think this adds additional complexity in sample. Not a strong opinion.

If you decide to keep it, we should add a comment for why we are creating type - To demonstrate updating default type for model and since there isn't a type that already exists we are creating one here.